presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -78,6 +78,14 @@ Spilling Properties
     aggregations and joins (inner and outer), so this property will not reduce memory usage required for
     window functions, sorting and other join types.
 
+``optimizer.reorder-windows``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ * **Type:** ``Boolean``
+ * **Default value:** ``true``
+ * **Description:** Allow reordering windows in order to put those with the same partitioning next to each other. This will sometimes decrease the number of repartitionings. This can also be specified on a per-query basis using the ``reorder_windows`` session property.
```

Comment:
Either here or in other places is wrong formatting. Please take a look how David formats documentation. 

Here you have:
```
* **Description:** text....
```
while in other places you have:
```
* **Description:** 

text....
```

Reply:
Notes about changes:

* All text under the property header (including the bullet list) is indented 4 spaces
* No *"Description"* bullet
* The "type" name is lowercase. Use `data size` or `duration` rather than `string` where appropriate.
* Don't put a space when writing data size value. For example, write `32MB`, not `32 MB`
* Add an *"Allowed values"* bullet for `string` types that only allow certain values
* Add a *"Minimum value"* or *"Maximum value"* bullet when appropriate